bridge_adaptivity/module/static/module/js/base_drag_drop.js

```diff
@@ -38,22 +38,28 @@ function get_ordering_elements_list(moveIndex, elIndex) {
     return newCollectionList
 }
 
+function change_move_url(moveUrl, element_dataset) {
+    // Change move url
+    elIndex = parseInt(element_dataset.index);
+    return moveUrl.replace("?", elIndex + "?");
+}
+
 // Function processing drop of the element
 function drop_handler(event, el) {
     event.preventDefault();
     let data = event.dataTransfer.getData("text/plain").split(','),
         elIndex = parseInt(el.dataset.index), // index of the target element
         moveUrl = data[1], // move URL of the dropped element
         moveIndex = parseInt(data[0]), // initial index of the dropped element
-    is_forbidden_extra_cheack = is_forbidden_to_chage(el.dataset, data)
+    is_forbidden_extra_cheack = is_forbidden_to_chage(el.dataset, data);
     if (moveIndex === elIndex || is_forbidden_extra_cheack) {
         console.log("Item doesn't change the order");
```

Comment:
Do we need this output?

Reply:
@arsentur 
Its need for logging the drag&drop process. If something wrong so we will able to find where this problem quickly.